### PR TITLE
helm: add `/config` proxy_pass to nginx gateway

### PR DESCRIPTION
**Pull request by LinTechSo**
### issue:
According to the issue https://github.com/grafana/loki/issues/9917
Adding Grafana Loki Enterprise to Grafana Apps Enterprise needs access to the GEL `/config` API path for cluster configuration

if GEL Apps in Grafana doesn't have an access to this path, client won't be able to see `/Cluster Configuration`  and also won't be able to create a data source in the Apps with the related Grafana Enterprise logs plugin.

### more information: 

`HOME/Apps/Grfana Enterprise Logs/Cluster Configuration`

```
A note about this cluster configuration page

The parameter values shown below are those returned by http://example.com:30519/config. If you have configured other components in your cluster with different parameter values, those will not be reflected below.

For documentation on the function of each configuration parameter, see [Configuration reference](https://grafana.com/docs/enterprise-logs/latest/config/reference/).
```
Screenshot 1402-04-30 at 03 17 34

This commit will add `/config` to nginx gateway on SSD mode loki helm chart deployment

### CODE REVIEW

**Comment by LinTechSo** [comment]
This commit will fix issue https://github.com/grafana/loki/issues/9917

**Review by chaudum — COMMENTED** [review]
LGTM

Could you please also add a changelog entry?

**Comment by LinTechSo** [comment]
Hi @chaudum .
thanks. it is done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -65,6 +65,7 @@
 * [9773](https://github.com/grafana/loki/pull/9773) **ssncferreira**: Fix instant query summary statistic's `splits` corresponding to the number of subqueries a query is split into based on `split_queries_by_interval`.
 * [9949](https://github.com/grafana/loki/pull/9949) **masslessparticle**: Fix pipelines to clear caches when tailing to avoid resource exhaustion.
 * [9936](https://github.com/grafana/loki/pull/9936) **masslessparticle**: Fix the way query stages are reordered when `unpack` is present.
+* [10000](https://github.com/grafana/loki/pull/10000) **LinTechSo**: helm: Add `/config` proxy_pass to nginx gateway
```
Comment (MichelHollands) [inline CHANGELOG.md:68]: This should be in a different CHANGELOG. For Helm chart changes bump the Helm chart version in production/helm/loki/Chart.yaml and update production/helm/loki/CHANGELOG.md and production/helm/loki/README.md. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

Comment (LinTechSo) [reply CHANGELOG.md]: Thanks for the help.

**Comment by MichelHollands** [comment]
@LinTechSo  Thanks for this contribution. Can you sign the CLA? Once that is done I'll merge this.

**Comment by LinTechSo** [comment]
>  thanks. it is done.

**Comment by MichelHollands** [comment]
@LinTechSo Can you rebase and change the version to 5.8.11? Another PR snuck in before this one.

**Comment by LinTechSo** [comment]
i fixed conflicts with chart version 5.8.10 by jump my PR chart version to 5.8.11 
is it OK ? @MichelHollands

**Review by MichelHollands — APPROVED** [review]

**Review by MichelHollands — APPROVED** [review]

**Pull request merged**